lib/page-generate.jsx

```diff
@@ -100,9 +101,10 @@ function run(location, el) {
     we will be using the value there otherwise we will default to
     what was assigned above.
   */
-  supportedLocale = Object.keys(locales).indexOf(navigator.language) !== -1;
-  currentLocale = supportedLocale ? navigator.language : currentLocale;
+  currentLocale = window.location.pathname.split('/')[1];
   messages = locales[currentLocale] || messages;
+  // Load React's data for the current locale, but if it's not available for that specific country, load the general language data
+  addLocaleData(window.ReactIntlLocaleData[currentLocale] || window.ReactIntlLocaleData[currentLocale.split('-')[0]]);
```

Comment:
`window.ReactIntlLocaleData` is an object and the key of the object is the locale name which is being loaded from `index.html` anything. It's a bit redundant here to try and figure out what locale where it's already been done in `index.html`.